### PR TITLE
A new tutorial for multiple choice questions

### PR DESCRIPTION
**Pull request by Zeyuzhao**
Add tutorial for multiple choice questions. Techniques mentioned:
* Zero-shot
* Reordering
* Adding additional context

### CODE REVIEW

File: docs/applied_prompting/mc_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 8
+locale: en-us
+style: chicago
+---
+
+# Multiple choice questions
+
+
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:8]: Add an introduction here, e.g. `Let's use PE to solve an LSAT question!...`

File: docs/applied_prompting/mc_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+This **could** be a result:
+
+
+```
+e) John of Worcester's account included a drawing of the sunspots, which could be the earliest illustration of sunspot activity.
+```
+
+<details scheme='warning'>
+<summary>Why is my answer different?</summary>
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:44]: set temp and top p to 0 in embed to avoid this

Comment (trigaten) [reply docs/applied_prompting/mc_tutorial.md:44]: User should be able to reproduce the result

File: docs/applied_prompting/mc_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+:::info
+Our model is autoregressive: prior text, the original prompt text and additional model-generated text, affects the next output. In this case, the step-by-step keyword makes the model more verbose. The model may choose to summarize a question, or synthesize new information from the question before supplying an answer.
+
+The specific term for this behavior is Chain of Thought: the model sequentially generates statements to reach an answer. This is similar to the concept of System 2 thinking (from Thinking Fast and Slow): the model defaults to system 1 thinking, but can chain system 1 thinking to arrive at a more methodological answer. 
+
+:::
+
+We can view the probabilities of the model like so:
+We can obtain probabilities of each predictions through the “show probabilities” tab:
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:75]: re word this

File: docs/applied_prompting/mc_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+If we add a bit of context, like so:
+
+```
+...
+Let's explain step by step. The formula for bayes is 
+```
+the model will approach using the right formula, Bayes.
+
+:::warning
+GPT model doesn't perform arithmetic operations well. You might notice that while the expression written is corrected, the computed number is not.
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:159]: Can you link to the MRKL page here :)

**Comment by trigaten** [comment]
Can you add a quick note/link to what the LSAT is

**Comment by trigaten** [comment]
Put full prompt instead of ...
It is slightly confusing

**Comment by trigaten** [comment]
Additional Context seems unrelated to LSAT. Pull this content together

**Comment by Zeyuzhao** [comment]
> Put full prompt instead of ... It is slightly confusing

Use "..." because it shortens the prompt. Or else, it is kind repetitive. We can have a "hide" functionality in the future?

File: docs/applied_prompting/mc_tutorial.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+---
+
+# Multiple choice questions
+
+Let's use GPT to solve an LSAT[^1] question!
+
+[^1]: 
+The LSAT (Law School Admission Test) is a standardized test used by law schools in the United States to assess the critical thinking and analytical reasoning skills of prospective students.
+
+Here is LSAT question:
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:13]: a*

File: docs/applied_prompting/mc_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
...s%20more%20than%201000%20years%20before%20John%20of%20Worcester%20did.%20%0Ac)%20Only%20heavy%20sunspot%20activity%20could%20have%20resulted%20in%20an%20aurora%20borealis%20viewable%20at%20a%20latitude%20as%20low%20as%20that%20of%20Korea.%0Ad)%20Because%20it%20is%20impossible%20to%20view%20sunspots%20with%20the%20naked%20eye%20under%20typical%20daylight%20conditions%2C%20the%20sighting%20recorded%20by%20John%20of%20Worcester%20would%20have%20taken%20place%20under%20unusual%20weather%20conditions%20such%20as%20fog%20or%20thin%20clouds.%0Ae)%20John%20of%20Worcester's%20account%20included%20a%20drawing%20of%20the%20sunspots%2C%20which%20could%20be%20the%20earliest%20illustration%20of%20sunspot%20activity.%0A&response=C)%20Only%20heavy%20sunspot%20activity%20could%20have%20resulted%20in%20an%20aurora%20borealis%20viewable%20at%20a%20latitude%20as%20low%20as%20that%20of%20Korea.&maxTokens=256&boxRows=3&uid=a4361398-8182-4fa3-b396-beadcbdbe9f3" frameBorder="0" style={{overflow:"hidden",width:"100%"}} width="100%" onLoad={e=>{let t=e.currentTarget;window.addEventListener("message",e=>{"frameheight-a4361398-8182-4fa3-b396-beadcbdbe9f3"==e.data[0]&&(t.height=e.data[1]+"px")},!1)}}></iframe>
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:36]: Can you increase the length of this embed?

File: docs/applied_prompting/mc_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+This **could** be a result:
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:38]: the result

File: docs/applied_prompting/mc_tutorial.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+This is not the right answer, and gives little insight into the “reasoning” of the decision. We can try adding the phrase `let's explain step by step` like so:
+
+
+```markdown
+...
+e) John of Worcester's account included a drawing of the sunspots, which could be the earliest illustration of sunspot activity.
+
+Let’s explain step by step
+```
+This phrase will increase verbosity of the model. You might get output like this:
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:61]: an output*

File: docs/applied_prompting/mc_tutorial.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+c) Only heavy sunspot activity could have resulted in an aurora borealis viewable at a latitude as low as that of Korea. 
+d) Because it is impossible to view sunspots with the naked eye under typical daylight conditions, the sighting recorded by John of Worcester would have taken place under unusual weather conditions such as fog or thin clouds. 
+e) John of Worcester's account included a drawing of the sunspots, which could be the earliest illustration of sunspot activity.
+```
+
+<details>
+<summary>The correct answer is ...</summary>
+c) Only heavy sunspot activity could have resulted in an aurora borealis viewable at a latitude as low as that of Korea. 
+</details>
+
```
Comment (trigaten) [inline docs/applied_prompting/mc_tutorial.md:32]: its already in the demo

**Comment by trigaten** [comment]
GG

**Pull request merged**